src/ajax/map_variants.php

```diff
@@ -198,6 +198,166 @@ function lovd_mapVariantToTranscripts (&$aVariant, $aTranscripts)
+                        // This variant is either not HGVS-compliant, not supported by our
+                        //  LOVD HGVS-check or it is not supported by VariantValidator.
+                        // Either way, we cannot perform lift overs for this one.
+                        continue;
+                    }
+
+                    if (!isset($_SETT['human_builds'][$sSourceBuild][$sChr])
+                        || !$_SETT['human_builds'][$sSourceBuild]['supported_by_VV']) {
+                        // This reference sequence is not supported by VariantValidator,
+                        //  so mapping is also out of the question.
+                        continue;
+                    }
+
+                    // Adding reference sequence to the variant description.
+                    $sFullVariant = $_SETT['human_builds'][$sSourceBuild][$sChr] . ':' . $sVariant;
+//                    $sFullVariant = (lovd_holdsRefSeq($sVariant)?
+//                        $sVariant :
+//                        $_SETT['human_builds'][$sSourceBuild][$sChr] . ':' . $sVariant
+//                    ); TODO: Activate this code once the necessary function has been pulled in.
+
+                    // Performing the lift over.
+                    $_VV = new LOVD_VV();
+                    $aVVResponse = ($_VV->verifyGenomicAndLiftOver($sVariant, array()));
+
+                    if ($aVVResponse == false
+                        || isset($aVVResponse[$sBuild])) {
+                        // Something went wrong... Possibly a network error?
+                        // These types of errors are not caused by the variant description
+                        //  itself. It might very well be a temporary problem. Therefore,
+                        //  we will add a try-again note to clarify that this variant might
+                        //  be fixable in the future.
+                        $bMappingTryAgain = true;
+                        continue;
+                    }
+
+                    if (!empty($aVVResponse['errors'])
+                        || empty($aVVResponse[$sBuild])) {
+                        // Something more serious went wrong.
+                        // TODO: Discuss these scenarios with Ivo:
+                        //  what situations are there and how do we want to respond?
+                        continue;
+                    }
+
+                    // Great! All checks have passed and the new description
+                    //  can be sent to the database.
+                    $aNewVariant = lovd_getVariantInfo($aVVResponse[$sBuild]);
+                    $_DB->query(
+                        'UPDATE ' . TABLE_VARIANTS .
+                        ' SET ' . $aGBColumns['DNA'] . ' = ?' .
+                        ', ' . $aGBColumns['position_start'] . ' = ?' .
+                        ', ' . $aGBColumns['position_end'] . ' = ?' .
+                        ', mapping_flags = mapping_flags | ' . MAPPING_DONE .
```

Comment:
Can we really add `MAPPING_DONE`? The lift overs are done, but what about the transcripts?

Reply:
Resolved! We do not want to add `MAPPING_DONE`.

Reply:
That is correct indeed, the flag should not be added.